x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -310,6 +311,25 @@ const getSavedObjectTypes = (
       },
     },
   },
+  [ASSETS_SAVED_OBJECT_TYPE]: {
+    name: ASSETS_SAVED_OBJECT_TYPE,
+    hidden: false,
+    namespaceType: 'agnostic',
+    management: {
+      importableAndExportable: false,
+    },
+    mappings: {
+      properties: {
+        package_name: { type: 'keyword' },
+        package_version: { type: 'keyword' },
+        install_source: { type: 'keyword' },
+        path: { type: 'text' },
+        media_type: { type: 'keyword' },
+        data_utf8: { type: 'text' },
```

Comment:
we probably do not want to index that (`index: false`)?

Reply:
thanks. see 19b9365